<?xml version="1.0"?>
<launch>
  <arg name="marker_size" default="50.0" />
  <arg name="max_new_marker_error" default="0.08" />
  <arg name="max_track_error" default="0.2" />
  <arg name="cam_image_topic" default="bebop/image_raw" />
  <arg name="cam_info_topic" default="bebop/camera_info" />
  <arg name="output_frame" default="camera" />
  <arg name="alvar_max_freq" default="30.0" />

  <!--node name="republish" pkg="image_transport" type="republish" args="compressed in:=bebop/image_raw raw out:=bebop/image_raw" output="screen" /-->

  <node name="ar_track_alvar" pkg="ar_track_alvar" type="individualMarkersNoKinect" respawn="false" output="screen" args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error) $(arg cam_image_topic) $(arg cam_info_topic) $(arg output_frame) $(arg alvar_max_freq)" />
  <node name="depth_eval_node" pkg="bebop_hri" type="depth_eval_node" output="screen" />
  <node pkg="twist_mux" type="twist_mux" name="twist_mux" output="screen">
    <remap from="cmd_vel_out" to="/bebop/cmd_vel"/>
    <rosparam file="$(find bebop_hri)/config/twist_mux.yaml" command="load"/>
    <rosparam file="$(find bebop_hri)/config/twist_mux_locks.yaml" command="load"/>
  </node>
</launch>
